BE/test.py

```
@@ -0,0 +1,52 @@
+from huggingface_hub import login
+from smolagents import CodeAgent, DuckDuckGoSearchTool, HfApiModel, tool
```

Ruff (0.8.2)

`smolagents.DuckDuckGoSearchTool` imported but unused

Remove unused import

(F401)